docs/spec/loops.md

```diff
@@ -90,7 +90,7 @@ In the example below, we are constructing a `subnets` property of a virtual netw
 variables subnets = [
```

Comment:
we also should change `variables` to `var` while we are editing this

Reply:
Yes we should - can't believe I missed it.

Reply:
Pushed out an update